packages/react-i18n/README.md

```diff
@@ -0,0 +1,80 @@
+# `@shopify/react-i18n`
+
+[![Build Status](https://travis-ci.org/Shopify/quilt.svg?branch=master)](https://travis-ci.org/Shopify/quilt)
+[![License: MIT](https://img.shields.io/badge/License-MIT-green.svg)](LICENSE.md) [![npm version](https://badge.fury.io/js/%40shopify%2Freact-i18n.svg)](https://badge.fury.io/js/%40shopify%2Freact-i18n.svg) [![npm bundle size (minified + gzip)](https://img.shields.io/bundlephobia/minzip/@shopify/react-i18n.svg)](https://img.shields.io/bundlephobia/minzip/@shopify/react-i18n.svg)
+
+I18n library for React
+
+## Installation
+
+```bash
+$ yarn add @shopify/react-i18n
+```
+
+## Usage
+
+Set up the top-level provider & locale manager:
```

Comment:
you need to introduce the parts of this library before referring to them. Assume people don't know what this p[ackage is.